apps/docs/src/app/core/component-docs/fixed-card-layout/examples/mobile/fixed-card-layout-mobile-examples.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectorRef, Component, ViewChild } from '@angular/core';
+import { FixedCardLayoutComponent } from '@fundamental-ngx/core';
+
+const portraitWidth = 385; // width in pixel for phone
+const landscapeWidth = 667;
```

Comment:
667 is not enought, we should take into account scrollbar width as well

Reply:
fixed.